.circleci/config.yml

```diff
@@ -7,23 +7,44 @@ jobs:
     steps:
       - checkout
       - run:
-         name: Install awscli
+         name: Installing awscli
          command: |
             apt-get -y -qq update
             apt-get -y -qq install awscli
             echo '*****AWS-CLI INSTALLED SUCCESSFULLY*****'
       - run: 
-          name: Build the codebase
+          name: Building the codebase
           command: ./scripts/build.sh
       - run: 
-          name: Deploying the codebase'
+          name: Deploying the codebase
           command: ./scripts/deploy-to-s3.sh
-
+  build_dev:
+    description: our build and deploy job for the website dev environment
+    docker:
+      - image: node:boron
+    steps:
+      - checkout
+      - run:
+         name: Installing awscli
+         command: |
+            apt-get -y -qq update
+            apt-get -y -qq install awscli
+            echo '*****AWS-CLI INSTALLED SUCCESSFULLY*****'
+      - run: 
+          name: Building the codebase
+          command: ./scripts/build.sh
+      - run: 
+          name: Deploying the codebase
+          command: sh ./scripts/deploy-dev-to-s3.sh
```

Comment:
consistency is important. Why are we calling the deploy script under the `sh` shell and the build script under `bash` shell?

Reply:
@eladg Got premmsion denied for this script so I added this. I will add it to the other files as well